asserts/fetcher.go

```diff
@@ -60,6 +63,34 @@ func NewFetcher(trustedDB RODatabase, retrieve func(*Ref) (Assertion, error), sa
 	}
 }
 
+type SeqFetcher interface {
+	// FetchSequence retrieves the assertion as indicated by its sequence reference.
+	FetchSequence(*AtSequence) error
+}
+
+// NewSeqFetcher creates a Fetcher which will use trustedDB to determine trusted
```

Comment:
this doc comment seems confusing,  trusted assertions are very likely not sequence forming. Shouldn't the first line mention SequenceFormingFetcher as that's the result type? The doc comment also doesn't talk about retrieveSeq...

Reply:
Updated this, thanks for the suggestion!